README.md

```diff
@@ -0,0 +1,60 @@
+# sbt launcher module
+
+This project is the componetized sbt launcher.   It can be used to launch many Maven/Ivy deployed applications
+and utilities, and forms the basis of [sbt](https://github.com/sbt/sbt),
+[activator](https://github.com/typesafehub/activator) and [conscript](https://github.com/n8han/conscript)'s launching
+abilities.
+
+For the full set of documentation, read: http://www.scala-sbt.org/0.13/docs/Sbt-Launcher.html
+.
+
+
+## Rebundling
```

Comment:
I think this section should move into somewhere under http://www.scala-sbt.org/0.13/docs/Sbt-Launcher.html.
Also it should be noted that rebundling is optional, and only for those who wants to redistribute the launcher.